diff -up kmediafactory-0.8.0/src/CMakeLists.txt.dso kmediafactory-0.8.0/src/CMakeLists.txt
--- kmediafactory-0.8.0/src/CMakeLists.txt.dso	2010-04-08 09:00:02.000000000 -0500
+++ kmediafactory-0.8.0/src/CMakeLists.txt	2010-06-14 15:15:31.337790709 -0500
@@ -35,7 +35,7 @@ qt4_add_dbus_adaptor(kmediafactory_SRCS 
 kde4_add_kcfg_files(kmediafactory_SRCS kmediafactorysettings.kcfgc)
 kde4_add_executable(kmediafactory ${kmediafactory_SRCS})
 if (KDE_4_4)
-target_link_libraries(kmediafactory kmf kmediafactoryinterfaces ${KDE4_KNEWSTUFF3_LIBS} ${QT_QTWEBKIT_LIBRARY})
+target_link_libraries(kmediafactory kmf kmediafactoryinterfaces ${KDE4_KNEWSTUFF3_LIBS} ${KDE4_KIO_LIBS} ${KDE4_THREADWEAVER_LIBRARIES} ${QT_QTWEBKIT_LIBRARY} ${X11_X11_LIB} )
 else (KDE_4_4)
 target_link_libraries(kmediafactory kmf kmediafactoryinterfaces ${KDE4_KNEWSTUFF2_LIBS} ${QT_QTWEBKIT_LIBRARY})
 endif (KDE_4_4)
