--- kmediafactory-0.8.1.orig/lib/videofile.cpp	2014-10-21 23:37:08.195109000 +1100
+++ kmediafactory-0.8.1/lib/videofile.cpp	2014-10-22 22:46:18.595957555 +1100
@@ -71,7 +71,7 @@
 #else
     if ( av_open_input_file(&avFromatContext, QFile::encodeName(filename).constData(), 0L, 0, 0L) != 0 ||
 #endif
-         av_find_stream_info(avFromatContext) < 0) {
+         avformat_find_stream_info(avFromatContext, NULL) < 0) {
         close();
     }
     else {
@@ -92,7 +92,7 @@
     }
 
     if (avFromatContext) {
-        av_close_input_file(avFromatContext);
+        avformat_close_input(&avFromatContext);
         avFromatContext = 0L;
     }
 
