diff -up kmediafactory-0.8.1/plugins/video/videoobject.cpp.gcc47 kmediafactory-0.8.1/plugins/video/videoobject.cpp
--- kmediafactory-0.8.1/plugins/video/videoobject.cpp.gcc47	2011-08-01 02:47:08.000000000 -0500
+++ kmediafactory-0.8.1/plugins/video/videoobject.cpp	2012-05-04 07:47:07.166222072 -0500
@@ -597,15 +597,15 @@ QVariant VideoObject::writeDvdAuthorXml(
             lang = "xx";
         }
 
-        QDomElement sub = doc.createElement("subpicture");
+        QDomElement sub2 = doc.createElement("subpicture");
 
         if ((lang == preferredLanguage) && (subFound == false)) {
             subFound = true;
             subTrack = i;
         }
 
-        sub.setAttribute("lang", lang);
-        titles.appendChild(sub);
+        sub2.setAttribute("lang", lang);
+        titles.appendChild(sub2);
         ++i;
     }
 
