diff -up kmediafactory-0.8.1/lib/videofile.cpp.ffmpeg kmediafactory-0.8.1/lib/videofile.cpp
--- kmediafactory-0.8.1/lib/videofile.cpp.ffmpeg	2011-08-01 02:47:08.000000000 -0500
+++ kmediafactory-0.8.1/lib/videofile.cpp	2012-10-22 06:29:32.797016215 -0500
@@ -58,7 +58,7 @@ bool VideoFile::open(const QString& file
     static bool init=false;
     if(!init) {
         av_register_all();
-        avcodec_init();
+        // avcodec_init();  // avodec_init has been deprecated since at least ffmpeg-0.10.x
         avcodec_register_all();
         av_log_set_level(0);
         init=true;
@@ -66,7 +66,11 @@ bool VideoFile::open(const QString& file
 
     close();
 
+#if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(53,2,0)
+    if ( avformat_open_input(&avFromatContext, QFile::encodeName(filename).constData(), 0L, 0) != 0 ||
+#else
     if ( av_open_input_file(&avFromatContext, QFile::encodeName(filename).constData(), 0L, 0, 0L) != 0 ||
+#endif
          av_find_stream_info(avFromatContext) < 0) {
         close();
     }
@@ -143,8 +143,11 @@ bool VideoFile::initialize()
     }
 
     avVideoCodecContext->workaround_bugs = 1;
-
+#if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(53,2,0)
+    return avcodec_open2(avVideoCodecContext, avVideoCodec, NULL)>=0;
+#else
     return avcodec_open(avVideoCodecContext, avVideoCodec)>=0;
+#endif
 }
 
 int VideoFile::getWidth()
